shivanand.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+BASE_URL = "https://reqres.in/api/users"
+
+def apiCall():
+    response = requests.get(BASE_URL)
+    print(response.json())
+
+apiCall()
+
+#this is feature branch 
```

Comment:
You dont have to call this method. This is not required. You can remove this.

Reply:
removed function call